### DISCUSSION TITLE
24-Hour Format Time poorly centered when hovered

### DISCUSSION THREAD

**Discussion by NotGumballer91** [ppy/osu#21313] [Q&A] [answered]
Is this intended?
![grafik](https://user-images.githubusercontent.com/64581009/202850650-d62a8360-494d-414f-9300-476688d4d10b.png)

**Comment by mk56-spn** [comment]
I assume you mean it being poorly centered ?

**Comment by frenzibyte** [comment]
That is going to be due to not using fixed-width for clocks (because it would look bad). Forcing it to be centered would end up moving the clock back and fourth depending on the numbers. Maybe increasing the width would alleviate the issue.

**Reply by peppy** [reply]
Adding a small amount of padding to the right might work well?

**Reply by frenzibyte** [reply]
Would make time combinations with lesser width look pushed to the left, not sure. Would have to check.

**Comment by NotGumballer91** [comment, marked as answer]
- This is fixed from #26167
